packages/web-app-files/src/mixins/spaces/actions/restore.js

```diff
@@ -0,0 +1,72 @@
+import { mapActions } from 'vuex'
+
+export default {
+  computed: {
+    $_restore_items() {
+      return [
+        {
+          name: 'restore',
+          icon: 'restart',
+          label: () => {
+            return this.$gettext('Restore')
+          },
+          handler: this.$_restore_showModal,
+          isEnabled: (space) => {
```

Comment:
Please use an array of resources for `isEnabled` and introduce a check that returns `false` if the number of provided resources is not 1.